### ISSUE TITLE
False positive "Unexpected indentation" for a chained scope function

### ISSUE THREAD

**Issue by artour-bakiev** [pinterest/ktlint#1935]
## Expected Behavior

The code fragment (below) formatted with the default IDE settings. It shouldn't raise "Unexpected indentation" error.

## Observed Behavior

`ktlint` reports the following errors:

```
test.kt:537:1: Unexpected indentation (16) (should be 12) (indent)
test.kt:538:1: Unexpected indentation (12) (should be 8) (indent)
```

## Steps to Reproduce

Code example:
```kotlin
    private fun test(ie: Abc) {
        a.getFiltered(ie)
            .firstOrNull { acc ->
                acc == true
            }?.let { xyz ->
                refresh(xyz)
            }
    }
```

`test.kt:537` corresponds to the `refresh(xyz)` line.

## Your Environment

* Version of ktlint used - 0.48.2
* Android Studio - Electric Eel | 2022.1.1 Patch 2
* A custom gradle task for Gradle 7.6
* Operating System and version - MacOS 13.3.1

Content of . editorconfig

```
[*.{kt,kts}]
indent_size=4
insert_final_newline=true
max_line_length=120
ktlint_standard_wrapping = disabled
ktlint_standard_import-ordering = disabled
```

**Comment by paul-dingemans** [comment]
This is solved in current [snapshot](https://pinterest.github.io/ktlint/install/snapshot-build/) of the upcoming `0.49` release. However, note that the `wrapping` rule will become a mandatory rule when you want to run the `indent` rule.
